odoo/modules/registry.py

```diff
@@ -327,6 +327,25 @@ def field_computed(self):
                                     model_name, ", ".join(field.name for field in fields))
         return computed
 
+    @lazy_property
+    def fields_with_relational_triggers(self):
```

Comment:
suggested docstring: `Return a set of fields that, when modified, cause relational fields to change, and thereby indirect field recomputations.`

Reply:
Also, please move this code below `field_triggers`.